cli-tests/t_encrypt_raw_key.sh

```diff
@@ -27,12 +27,27 @@ show_status()
 	fi
 }
 
-begin "Encrypt with raw_key protector"
+begin "Encrypt with raw_key protector from file"
 head -c 32 /dev/urandom > "$raw_key_file"
 fscrypt encrypt --quiet --name=prot --source=raw_key --key="$raw_key_file" "$dir"
 show_status true
 
-begin "Try to encrypt with raw_key protector, using wrong key length"
+begin "Encrypt with raw_key protector from stdin"
+head -c 32 /dev/urandom | fscrypt encrypt --quiet --name=prot --source=raw_key "$dir"
+show_status true
+
+begin "Try to encrypt with raw_key protector from file, using wrong key length"
 head -c 16 /dev/urandom > "$raw_key_file"
 _expect_failure "fscrypt encrypt --quiet --name=prot --source=raw_key --key='$raw_key_file' '$dir'"
 show_status false
+
+begin "Try to encrypt with raw_key protector from stdin, using wrong key length"
+_expect_failure "head -c 16 /dev/urandom | fscrypt encrypt --quiet --name=prot --source=raw_key '$dir'"
+show_status false
+
+begin "Encrypt with raw_key protector from file, unlock from stdin"
+head -c 32 /dev/urandom > "$raw_key_file"
+fscrypt encrypt --quiet --name=prot --source=raw_key --key="$raw_key_file" "$dir"
+fscrypt lock "$dir"
+cat "$raw_key_file" | fscrypt unlock --quiet "$dir"
```

Comment:
This line is causing a shellcheck warning.  Can you change it to:

```
   fscrypt unlock --quiet "$dir" < "$raw_key_file"
```

Otherwise this pull request looks good to go, thanks!

Reply:
Potayto... Potahto... The bots are so picky nowadays... 😝 OK, I've rebased and force-pushed, but haven't tested it. The bot is better be right. 🤣